dbs/postgres/latest/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:11-alpine
+FROM postgres:latest
```

Comment:
Why are we abandoning apline?

Reply:
postgres doesn't have a latest-alpine tag unfortunately, so I'm relying on the standard image